src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+header {
```

Comment:
Get used to style all elements using classes

Reply:
I'm sorry, I thought the header and body don't need classes named in the same way..?